src/admin/openapi.yaml

```diff
@@ -64,6 +64,9 @@ tags:
   - name: Expenses
     description: |
       This API gives you access to your organisation's Expenses
+  - name: Dependent Expenses Field Values
```

Comment:
Don't create a top-level item for this.. this can be inside Expenses which is where Expense Field APIs live as well